app/Models/Sensor.php

```diff
@@ -171,6 +171,18 @@ public function currentStatus(): Severity
         return Severity::Ok;
     }
 
+    public function hasTresholds(): bool
```

Comment:
Should this be hasThresholds?

Reply:
ah shoot, I scanned right over that.  Good catch.